### PR TITLE
feat(subject): tags

### PR DESCRIPTION
**Pull request by Jarvie8176**
https://github.com/bangumi/server/issues/42

### CODE REVIEW

File: pol/api/v0/models/subject.py [hunk 1]
```diff
@@ -51,6 +56,8 @@ class Subject(BaseModel):
 
     collection: Collection
 
+    tags: List[Tags]
```
Comment (Trim21) [inline pol/api/v0/models/subject.py:59]:
```suggestion
    tags: List[Tag]
```

File: pol/api/v0/models/subject.py [hunk 2]
```diff
@@ -29,6 +29,11 @@ class Collection(BaseModel):
     dropped: int
 
 
+class Tags(BaseModel):
+    tag_name: str
```
Comment (Trim21) [inline pol/api/v0/models/subject.py:33]:
```suggestion
    name: str
```

File: pol/api/v0/models/subject.py [hunk 3]
```diff
@@ -29,6 +29,11 @@ class Collection(BaseModel):
...
+    result: int
```
Comment (Trim21) [inline pol/api/v0/models/subject.py:34]:
```suggestion
    count: int
```

File: pol/curd/subject.py [hunk 4]
```diff
@@ -50,6 +51,9 @@ class Subject(BaseModel):
     rate_9: int = Field(alias="field_rate_9")
     rate_10: int = Field(alias="field_rate_10")
 
+    # 序列化标签
+    field_tags: str = Field(alias="field_tags")
```
Comment (Trim21) [inline pol/curd/subject.py:55]: 这就没必要加alias了

File: tests/app/api_v0/test_subject.py [hunk 5]
```diff
@@ -128,3 +132,65 @@ def test_subject_cache_broken_purge(client: TestClient, redis_client: Redis):
...
+        {"tag_name": "冷泉夜月", "result": 1},
+        {"tag_name": "银他妈", "result": 1},
+        {"tag_name": "陈老师", "result": 1},
+        {"tag_name": "银魂", "result": 1},
+        {"tag_name": "治愈系", "result": 1},
+        {"tag_name": "恶搞", "result": 1},
+    ]
+
+
+def test_subject_tags_empty(client: TestClient, db_session: Session):
```
Comment (Trim21) [inline tests/app/api_v0/test_subject.py:153]: 用这个fixture
https://github.com/bangumi/server/blob/2b569560cecf3f331a0c44d3722520a03e8b00dd/tests/conftest.py#L58

**Review by Trim21 — CHANGES_REQUESTED** [review]

File: pol/curd/subject.py (hunk 4 above)
Comment (Jarvie8176) [reply pol/curd/subject.py:55]: 确实，我换个好点的名字

File: tests/app/api_v0/test_subject.py (hunk 5 above)
Comment (Jarvie8176) [reply tests/app/api_v0/test_subject.py]: 这里的`subject_id` hardcode成200了，盲猜是个bug？

https://github.com/bangumi/server/blob/2b569560cecf3f331a0c44d3722520a03e8b00dd/tests/conftest.py#L92

**Pull request merged**